plugins/vendor/zoom/lib/model/oAuth/kZoomOauth.php

```diff
@@ -117,6 +118,19 @@ public static function verifyHeaderToken($zoomConfiguration)
 			}
 		}
 
+		if(isset($headers[self::X_ZM_SIGNATURE]) && isset($headers[self::X_ZM_REQUEST_TIMESTAMP]))
+		{
+			$signatureTimestamp = $headers[self::X_ZM_REQUEST_TIMESTAMP];
+			$secretToken = $zoomConfiguration[kOAuth::SECRET_TOKEN];
+			$body = file_get_contents(ZoomHelper::PHP_INPUT);
```

Comment:
align so we will have one getPayloadData function that have option to get the data with/without json decode